index.bs

```diff
@@ -54,6 +54,7 @@ urlPrefix: https://html.spec.whatwg.org/multipage/
             text: rules for extracting the chapter title
             text: text track cue start time
             text: text track cue end time
+            text: unbounded text track cue
```

Comment:
Turns out this term isn't linked any longer, so this line could be removed. Or maybe a note could be added saying that a cue with `endTime` Infinity is an unbounded text track cue, linking to it.

Reply:
Well spotted. I'll either add a note as you suggest or link to https://html.spec.whatwg.org/multipage/media.html#unbounded-text-track-cue.

Reply:
Added note with a link to [unbounded text track cue](https://html.spec.whatwg.org/multipage/embedded-content.html#unbounded-text-track-cue), though I'm unsure whether my wording is correct as I was unable to find a comparable example.

Reply:
Sounds like reasonable wording to me.